tests/loaders/test_dsv_loader.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from unittest.mock import MagicMock
+from data_lakehouse_ingest.loaders.dsv_loader import load_csv_data, load_tsv_data
+
+@pytest.fixture
+def mock_spark():
+    spark = MagicMock()
```

Comment:
You can make a `conftest.py` file that holds fixtures that are used in numerous tests to prevent code duplication.